Project.toml

```
@@ -1,8 +1,9 @@
 name = "Setfield"
 uuid = "efcf1570-3423-57d1-acb7-fd33fddbac46"
-version = "0.4.1"
+ version = "0.5.0"
```

```suggestion
version = "0.5.0"
```

Hmm... It looks like GitHub introduces a leading space when I use the suggestion feature?